crates/relayer/src/worker/map.rs

```diff
@@ -237,5 +237,6 @@ fn metric_type(o: &Object) -> ibc_telemetry::state::WorkerType {
         Object::Channel(_) => WorkerType::Channel,
         Object::Packet(_) => WorkerType::Packet,
         Object::Wallet(_) => WorkerType::Wallet,
+        Object::CrossChainQueryPacket(_) => WorkerType::CrossChainQueryPacket,
```

Comment:
Let's rename `Object::CrossChainQueryPacket` to `Object::CrossChainQuery` and `WorkerType::CrossChainQueryPacket` to `WorkerType::CrossChainQuery`